data/sheragi outfits.txt

```diff
@@ -0,0 +1,213 @@
+# Copyright (c) 2020 by Fernando S. Sáez
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# Outfits
+
+outfit "Electromagnetic Battery"
+	category "Power"
+	cost 1000000
+	thumbnail "outfit/embattery"
+	"mass" 150
+	"outfit space" -150
+	"shield generation" 0.8
+	"shield energy" 0.8
+	"shield heat" 0.8
+	"energy capacity" 20000
+	description `This is massive superconductive coil, capable of storing large amounts of energy in a magnetic field, and capable of discharging all of it in a fraction of a second. The magnetic field also provides a small boost to the shield regeneration of the ship, which is otherwise very underdeveloped.`
```

Comment:
```suggestion
	description `This is a massive superconductive coil, capable of storing large amounts of energy in a magnetic field and discharging all of it in a fraction of a second. The magnetic field also provides a small boost to the shield regeneration of the ship, which is otherwise very underdeveloped.`
```